vault/seal/ocikms/ocikms.go

```diff
@@ -0,0 +1,305 @@
+// Copyright © 2019, Oracle and/or its affiliates.
+package ocikms
+
+import (
+	"context"
+	b64 "encoding/base64"
```

Comment:
I'd remove this. `base64.*` is pervasive in Vault, and seeing b64 in code could cause confusing, miss grep matches, etc.